packages/mobile/src/store/auth/reducer.ts

```diff
@@ -1,13 +1,19 @@
 import { createReducer } from '@reduxjs/toolkit';
 import { DataStatus } from '~/common/enums/enums';
-import { signUp } from './actions';
+import { signUp, signIn } from './actions';
 
-type State = {
+export type State = {
```

Comment:
Let's move it as `export type { State }` at the end of the file

Reply:
corrected